### ISSUE TITLE
video_player does not give access to the file resolution?

### ISSUE THREAD

**Issue by stephane-archer** [flutter/flutter#147471]
### Use case

```
VideoPlayerController controller = new VideoPlayerController.file(myfile);
controller.value.duration ; // OK
controller.value.size ; // OK
controller.value.resolution; // is not working
```

### Proposal

up

**Comment by moffatman** [comment]
Isn't that just size? What do you mean by resolution?

**Comment by stephane-archer** [comment]
@moffatman Sorry, you are probably right but I found `size.height` and `size.width` to be equal to zero so I'm not sure what I'm missing

**Comment by moffatman** [comment]
Size should be there after you `initialize()` the controller.

**Comment by stephane-archer** [comment]
@moffatman thank you so much you are the best

tldr:
```
VideoPlayerController controller = VideoPlayerController.file(myFile);
await controller.initialize();
print(controller.value.size.height);
print(controller.value.size.width);
```
